docs/application/dotnet/guides/nui/layouting.md

```diff
@@ -0,0 +1,292 @@
+# Layouting
+
+This tutorial describes the NUI Layouting framework, covering the following subjects:
+
+[Overview](#overview)<br>
+[Setting Layouts](#settingLayouts)<br>
+[Sizing](#sizingLayouts)<br>
+[Padding/Margins](#paddingAndMargins)<br>
+[Minimum/Maximum sizes](#minimumAndMaximum)<br>
+[Common Layouts](#commonLayout)<br>
```

Comment:
docs.tizen.org automatically generates TOC on the right side. 
Please remove TOC.

Reply:
Okay, Removed TOC.